# HG changeset patch
# User Torsten <mttl@mailbox.org>
# Date 1518805015 -3600
#      Fri Feb 16 19:16:55 2018 +0100
# Node ID b96c76aed31e08e13b1a1a041bb0c45cc033ee64
# Parent  860860e2e69e4f3b3ea88f5f119d93dd62aa26d6
* webinfo.cc (close_tab): set focus to current tab after a tab was closed

diff --git a/libgui/src/qtinfo/webinfo.cc b/libgui/src/qtinfo/webinfo.cc
--- a/libgui/src/qtinfo/webinfo.cc
+++ b/libgui/src/qtinfo/webinfo.cc
@@ -283,16 +283,18 @@ namespace octave
         QWidget *w = static_cast<QWidget *> (tab_data.value<void*> ());
         _stacked_widget->removeWidget (w);
         delete w;
 
         _tab_bar->removeTab (index);
       }
 
     tab_state_changed ();
+
+    setFocus ();
   }
 
   void webinfo::load_ref (const QString& ref_name)
   {
     // Will load "Top", if ref_name was not found.
     load_node (_parser.find_reference (ref_name));
 
     if (_text_browser)
